packages/ember-glimmer/lib/environment.js

```diff
@@ -8,7 +8,7 @@ import { assert } from 'ember-metal/debug';
 import { CurlyComponentSyntax, CurlyComponentDefinition } from './syntax/curly-component';
 import { DynamicComponentSyntax } from './syntax/dynamic-component';
 import { OutletSyntax } from './syntax/outlet';
-import lookupComponent from './utils/lookup-component';
+import lookupComponent from 'ember-templates/utils/lookup-component';
```

Comment:
this is a circular package ref to the pseudo package that is just about reexports, shared stuff should go into ember-views or we can symlink for now.  I want to try to avoid imports from ember-glimmer and ember-htmlbars to ember-templates